### DISCUSSION TITLE
Botão de Notificação Via WhatsApp não aparece no meu painel

### DISCUSSION THREAD

**Discussion by MiqueiasMarinho** [RamonSilva20/mapos#1679] [Q&A]
Instalei a última versão do MAPos, porém, a opção de enviar notificação Via WhatsApp não aparece no meu painel. Aparece somente enviar via E-mail. Sabem me dizer o porque disso e como resolver?

**Comment by Pr3d4dor** [comment]
@MiqueiasMarinho configurou todas as informações no emitente?

**Comment by MiqueiasMarinho** [comment]
Opa... Obrigado pela resposta meu amigo. Olha o vídeo abaixo. Acho que configurei tudo certo.

https://www.loom.com/share/f0f5bffb6bd94c878f0f4f594b414ed5?focus_title=1&muted=1&from_recorder=1
________________________________
De: Gianluca Bine ***@***.***>
Enviado: sexta-feira, 4 de março de 2022 02:18
Para: RamonSilva20/mapos ***@***.***>
Cc: MiqueiasMarinho ***@***.***>; Mention ***@***.***>
Assunto: Re: [RamonSilva20/mapos] Botão de Notificação Via WhatsApp não aparece no meu painel (Discussion #1679)


@MiqueiasMarinho<https://github.com/MiqueiasMarinho> configurou todas as informações no emitente?

—
Reply to this email directly, view it on GitHub<https://github.com/RamonSilva20/mapos/discussions/1679#discussioncomment-2292643>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AYBFEQNDNYNK6Y2FH4BCQVDU6GFITANCNFSM5PZDE6BQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by willph** [comment]
Cadastro do cliente está preenchido o número de telefone?

Repete mesmo número em celular

Em sex, 4 de mar de 2022 13:04, MiqueiasMarinho ***@***.***>
escreveu:

**Comment by MiqueiasMarinho** [comment]
Puts.. era isso... Disfarça. Valeu mesmo, campeão.
________________________________
De: Wilmerson ***@***.***>
Enviado: sexta-feira, 4 de março de 2022 14:33
Para: RamonSilva20/mapos ***@***.***>
Cc: MiqueiasMarinho ***@***.***>; Mention ***@***.***>
Assunto: Re: [RamonSilva20/mapos] Botão de Notificação Via WhatsApp não aparece no meu painel (Discussion #1679)


Cadastro do cliente está preenchido o número de telefone?

Repete mesmo número em celular

Em sex, 4 de mar de 2022 13:04, MiqueiasMarinho ***@***.***>
escreveu:

> Opa... Obrigado pela resposta meu amigo. Olha o vídeo abaixo. Acho que
> configurei tudo certo.
>
>
> https://www.loom.com/share/f0f5bffb6bd94c878f0f4f594b414ed5?focus_title=1&muted=1&from_recorder=1
> ________________________________
> De: Gianluca Bine ***@***.***>
> Enviado: sexta-feira, 4 de março de 2022 02:18
> Para: RamonSilva20/mapos ***@***.***>
> Cc: MiqueiasMarinho ***@***.***>; Mention ***@***.***>
> Assunto: Re: [RamonSilva20/mapos] Botão de Notificação Via WhatsApp não
> aparece no meu painel (Discussion #1679)
>
>
> @MiqueiasMarinho<https://github.com/MiqueiasMarinho> configurou todas as
> informações no emitente?
>
> —
> Reply to this email directly, view it on GitHub<
> https://github.com/RamonSilva20/mapos/discussions/1679#discussioncomment-2292643>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/AYBFEQNDNYNK6Y2FH4BCQVDU6GFITANCNFSM5PZDE6BQ
> >.
> Triage notifications on the go with GitHub Mobile for iOS<
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android<
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> >.
> You are receiving this because you were mentioned.Message ID: ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/RamonSilva20/mapos/discussions/1679#discussioncomment-2296110>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEDNYMRVRARDYR6BYYFU4KDU6IYALANCNFSM5PZDE6BQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

—
Reply to this email directly, view it on GitHub<https://github.com/RamonSilva20/mapos/discussions/1679#discussioncomment-2296334>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AYBFEQNOLRGRRG4GMDJTK2TU6I3MTANCNFSM5PZDE6BQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
